hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -0,0 +1,447 @@
+apiVersion: v1
+kind: Template
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  required: true
+- name: IMAGE_TAG
+  required: true
+- name: REPO_NAME
+  value: route-monitor-operator
+  required: true
+metadata:
+  name: selectorsyncset-template
+objects:
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: bundle-manifests
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+    resourceApplyMode: Sync
+    resources:
```

Comment:
We need to also install the namespace

Reply:
We install the operator in the OpenShift monitoring namespace, so I am not sure we need to

Reply:
Ok yeah, if the namespace is always going to exist that is fine